cmd/gadgetctl/main.go

```diff
@@ -75,7 +78,20 @@ func main() {
 			fmt.Fprintf(os.Stderr, "Error: %v\n", err)
 			os.Exit(1)
 		}
+		gadgetctlVersion := version.Version()
 		runtime.InitDeployInfo()
+		info, err := deployinfo.Load()
+		if err != nil {
+			log.Warnf("failed to load deploy info: %s", err)
+		}
+		serverSemver, _ := semver.Make(info.ServerVersion)
+
+		if !gadgetctlVersion.Equals(serverSemver) {
+			log.Warnf(
+				"WARNING: version skew detected (gadget pods v%s vs gadgetctl v%s), use 'gadgetctl' to fix it.\n",
```

Comment:
Same about warning and line break. Additionally:

```suggestion
			log.Warnf(
				"Version skew detected (gadget daemon v%s vs gadgetctl v%s)",
```